src/pointer-analysis/dynamic_object_name.h

```diff
@@ -0,0 +1,57 @@
+/*******************************************************************\
+
+Module: Value Set
```

Comment:
Module: Dynamic object name
Purpose: To generate a name for dynamic objects suitable for use in the LHS of value set maps.